docs/changelog.rst

```diff
@@ -15,6 +15,62 @@ IN DEVELOPMENT
 
 Changes will be summarized here periodically.
 
+Version 2.10.2 (Dec 20, 2022)
+-----------------------------
+
+Changes:
+
+* ``scoped_interpreter`` constructor taking ``PyConfig``.
+  `#4330 <https://github.com/pybind/pybind11/pull/4330>`_
+
+* ``pybind11/eigen/tensor.h`` adds converters to and from ``Eigen::Tensor`` and
+  ``Eigen::TensorMap``.
+  `#4201 <https://github.com/pybind/pybind11/pull/4201>`_
+
+* Add option for enable/disable enum members in docstring.
+  `#2768 <https://github.com/pybind/pybind11/pull/2768>`_
+
+Bug fixes:
+
+
+* Bug fix affecting only Python 3.6 under very specific, uncommon conditions:
+  move ``PyEval_InitThreads()`` call to the correct location.
+  `#4350 <https://github.com/pybind/pybind11/pull/4350>`_
+
+* Fixed typing of ``KeysView``, ``ValuesView`` and ``ItemsView`` in ``bind_map``.
+  `#4353 <https://github.com/pybind/pybind11/pull/4353>`_
+
+* ``PyGILState_Check()``'s  were integrated to ``pybind11::handle``
+  ``inc_ref()`` & ``dec_ref()``. The added GIL checks are guarded by
+  ``PYBIND11_ASSERT_GIL_HELD_INCREF_DECREF``, which is the default only if
+  ``NDEBUG`` is not defined.
+  `#4246 <https://github.com/pybind/pybind11/pull/4246>`_
+
+* Fix segfault bug when passing foreign native functions to functional.h.
```

Comment:
@Lalaland for comment:

IIUC strictly speaking the segfault bug is only fixed `if PYBIND11_INTERNALS_VERSION > 4`, which is not the default.

I'm not sure if there are more nuances, and what makes sense to explain here.

Reply:
The segfault bug is partially fixed here for abi version 4. Remember, I added two unit tests for the bugfix that both failed under the old code

One of the unit tests now passes all the time. The other unit test only passes with the new abi version.

Reply:
I would leave this entry as is, and then have a new entry "More functional.h segfault fixes" when we do an ABI upgrade